Frontend/Gmfctn/src/app/core/interceptors/http/main-http.interseptor.ts

```diff
@@ -0,0 +1,39 @@
+import { HttpErrorResponse, HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { Observable, of, throwError } from 'rxjs';
+import { catchError, switchMap } from 'rxjs/operators';
+import { AuthenticateService } from 'src/app/modules/+auth/services/authenticate.service';
+
+
+@Injectable()
+export class MainHttpInterceptor implements HttpInterceptor {
+  constructor(private authenticateService: AuthenticateService, private router: Router) { }
+
+  private handleAuthError(err: HttpErrorResponse): Observable<any> {
+    if (err.status === 401 || err.status === 403) {
+        this.authenticateService.isErrorInAuthorization$.next(true);
+        this.router.navigateByUrl(`/auth/sign-in`);
+        return of(err.message);
```

Comment:
Missing space before return. It's applicable to the rest of the PR also

Reply:
Fixed